packages/playground/blueprints/src/lib/steps/set-site-language.ts

```diff
@@ -0,0 +1,23 @@
+import { StepHandler } from '.';
+
+export interface SetSiteLanguageStep {
+	step: 'setSiteLanguage';
+	/** The language to set, e.g. 'en_US' */
+	language: string;
+}
+
+export const setSiteLanguage: StepHandler<SetSiteLanguageStep> = async (
+	playground,
+	{ language }
+) => {
+	const docroot = await playground.documentRoot;
+
+	await playground.defineConstant('WPLANG', language);
+	await playground.run({
+		code: `<?php
+		require_once('${docroot}/wp-load.php');
+		require_once('${docroot}/wp-admin/includes/admin.php');
+		require_once('${docroot}/wp-admin/includes/translation-install.php');
+		wp_download_language_pack('${language}');`,
```

Comment:
This is a blocking download that only starts when we get to the `setSiteLanguage` step. Let's instead detect it early and add it to the resources queue like we do with wp-cli.phar here:

https://github.com/WordPress/wordpress-playground/blob/9af8499a09686139ff13d8e48e7f36cf0800854f/packages/playground/blueprints/src/lib/compile.ts#L173

I'd like that logic to live in the step handler once we migrate to PHP Blueprints, but here we are today.

Reply:
Thanks! This helped me a lot because I need access to other data from the blueprint (wp version, plugins and themes).